pkgs/development/libraries/libguytools/default.nix

```
@@ -0,0 +1,54 @@
+{ lib
+, stdenv
+, fetchurl
+, wrapQtAppsHook
+, qmake
+, qtbase
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "libguytools";
+  version = "2.1.0";
+
+  src = fetchurl {
+      url = "mirror://sourceforge/project/libguytools/libguytools/LatestSource/tools-${finalAttrs.version}.tar.gz";
+      hash = "sha256-eVYvjo2wKW2g9/9hL9nbQa1FRWDMMqMHok0V/adPHVY=";
+  };
+
+  qmakeFlags = [
+    "trunk.pro"
+    "toolsstatic.pro"
+  ];
+
+  enableParallelBuilding = true;
+
+  nativeBuildInputs = [ qmake wrapQtAppsHook ];
+  dontWrapQtApps = true;
+  buildInputs = [ qtbase ];
+
+  postPatch = ''
+    sed -i "/dpkg-buildflags/d" tools.pro
+    patchShebangs create_version_file.sh
+  '';
+
+  preConfigure = ''
+    ./create_version_file.sh
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out
+    cp -r lib $out/
+    cp -r include $out/
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "A small programming toolbox";
+    mainProgram = "libguytools";
+    homepage = "https://libguytools.sourceforge.io";
+    maintainers = with maintainers; [ d3vil0p3r ];
+    platforms = platforms.unix;
+    license = licenses.gpl2;
```

According to the [file header](https://sourceforge.net/p/libguytools/code/HEAD/tree/tags/tools-2.1.0/toolerror.cpp#l20) is the license `GPLv2+`. 

```suggestion
    license = licenses.gpl2Plus;
```